jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -268,6 +269,20 @@ class EndpointK @JvmOverloads constructor(
 
     override fun requestKeyframe(mediaSsrc: Long) = _transceiver.requestKeyFrame(mediaSsrc)
 
+    override fun isOversending(): Boolean = bitrateController.isOversending()
+
+    override fun setSelectedEndpoints(selectedEndpoints: MutableList<String>) {
```

Comment:
Why is this `MutableList` now rather than `List`?

Reply:
This was probably an auto-convert, but `MutableList` is the Kotlin equivalent of Java's list.  I agree it can be `List` though.

Reply:
Ah, no actually it's because it's an override of a java method which takes a Java `List`, but weirdly it does seem to let me override it with a Kotlin `List`.  I think I will leave it as a `MutableList` to be consistent with the Java parent, but add a TODO that it should become `List` once the java parent goes away.

Reply:
Ok.